packages/components/src/search-control/types.ts

```diff
@@ -30,26 +42,22 @@ export type SearchControlProps = Pick<
 	 * rather than just clearing the input value.
 	 */
 	onClose?: () => void;
+	/** @ignore */
+	onDrag?: InputControlProps[ 'onDrag' ];
+	/** @ignore */
+	onDragStart?: InputControlProps[ 'onDragStart' ];
+	/** @ignore */
+	onDragEnd?: InputControlProps[ 'onDragEnd' ];
```

Comment:
We need to specifically inherit these three event handler types from InputControl because they are different from standard `<input>` event handlers. However, they are irrelevant to SearchControl so we should hide them from the docs.

Reply:
TIL the `@ignore` tag